### ISSUE TITLE
Feature request - Shortcut to detach a tab

### ISSUE THREAD

**Issue by resting** [babyman/quick-tabs-chrome-extension#372]
Is there a shortcut to detach a tab to a new window?

Hopefully it'll be added if not available yet.

Thanks.

**Comment by MartinLichtblau** [comment]
No, there isn't. But there is an extensive extension for just that called [Tab to Window/Popup - Keyboard Shortcut ](https://chrome.google.com/webstore/detail/tab-to-windowpopup-keyboa/adbkphmimfcaeonicpmamfddbbnphikh).
